react-components/src/lib.js

```diff
@@ -1,15 +1,32 @@
 import * as apps from "./apps";
 import * as data from "./data";
+import * as util from './util/util';
+import palette from "./util/CyVersePalette";
 import React from "react";
 import ReactDOM from "react-dom";
-import {MuiThemeProvider, createMuiTheme} from "@material-ui/core/styles"; // v1.x
+import { MuiThemeProvider, createMuiTheme } from "@material-ui/core/styles"; // v1.x
 
 
 // gwt-react needs React and ReactDOM on the global object
 window.React = React;
 window.ReactDOM = ReactDOM;
 
 
-const getDefaultTheme = () => createMuiTheme({});
+const getDefaultTheme = () => createMuiTheme({
+    palette: {
+        primary: {
+            main: palette.blue,
+        },
+        secondary: {
+            main: palette.lightGreen
```

Comment:
How does `lightBlue` look for `secondary`?

The `lightGreen` reminds me of the [cyverse-ui](https://cyverse.github.io/cyverse-ui/) `success` color, but I'm not sure how much of that color palette is going to carry over to the MUI v1.x upgrade.

Reply:
Oh right, cyverse-ui looks to be using `lightBlue`.  I didn't see us using `secondary` anywhere so I just arbitrarily picked `lightGreen`.  I'll switch it though 👍